test/Image2.rb

```diff
@@ -10,46 +10,46 @@ class Image2_UT < Test::Unit::TestCase
     FreezeError = RUBY_VERSION[/^1\.9|^2/] ? RuntimeError : TypeError
 
     def setup
-        @img = Magick::Image.new(20, 20)
+      @img = Magick::Image.new(20, 20)
     end
 
     def test_composite!
-        img1 = Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
-        img2 = Magick::Image.read(IMAGES_DIR+'/Button_1.gif').first
-        assert_nothing_raised do
-            res = img1.composite!(img2, Magick::NorthWestGravity, Magick::OverCompositeOp)
-            assert_same(img1, res)
-        end
-        img1.freeze
-        assert_raise(FreezeError) { img1.composite!(img2, Magick::NorthWestGravity, Magick::OverCompositeOp) }
+      img1 = Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.